src/Sarif.Driver/Sdk/AnalyzeOptionsBase.cs

```diff
@@ -119,6 +119,6 @@ public abstract class AnalyzeOptionsBase : CommonOptionsBase
             "max-file-size-in-kb",
             HelpText = "The maximum file size (in kilobytes) that will be analyzed.",
             Default = 1024)]
```

Comment:
>1024 [](http://example.com/codeflow?start=22&length=4)

I also have a question here, we probably have a context so we added this parameter.

The other way is,
unless user specifically limit it, we don't put limit.
If so the default can be int.Max.

Or a reasonable bigger default value like 128MB.

Let me know if you like to change it and what value I can update the PR.

Reply:
I think changing the default would be a breaking change. So perhaps just making the setter public is the right fix.

Reply:
I think changing the default would be a breaking change --- Yes!